component/rbac.libsonnet

```diff
@@ -6,6 +6,28 @@ local inv = kap.inventory();
 
 local params = inv.parameters.openshift4_authentication;
 
+local sudoGroups =
+  if params.sudoGroupName != null && params.sudoGroupName != '' then
+    [
+      std.trace(
+        (
+          '\nParameter `sudoGroupName` is deprecated.\n' +
+          'Please update your config to use `sudoGroups` instead.'
+        ),
+        params.sudoGroupName
+      ),
+    ]
+  else
+    com.renderArray(params.sudoGroups);
```

Comment:
Couldn't we merge these by default? Would prevent a case where someone sets the legacy sudogroupname and invalidates the array set somewhere else

Reply:
I though it's complicated to do. But `sudoGroups` can easily be prepended since renderArray already handles duplicates.

Fixed.